### DISCUSSION TITLE
iBBQ-4T support

### DISCUSSION THREAD

**Discussion by lhassell** [make-all/tuya-local#826] [Q&A]
I see that support was added for the Inkbird iBBQ-4T. I attempted to add it today, but the only thing that is detected is the power indicator. How can I get the integration to add the remaining DPs (probe temps, temp alarms, etc...) Is there some way to tell the integration to look for the rest of the DPs? I can provide any needed info from the Tuya backend. Any help is appreciated.

**Comment by make-all** [comment]
The last comment on #750 suggests that turning the thermometer off then on again can help.

I am not sure what you mean by "the only thing that is detected is the power indicator".  Maybe some logs or a screenshot is needed to help me understand what the problem is.
